tools/IoTEdgeDevOps/DevOpsLib/BugPriority.cs

```diff
@@ -0,0 +1,11 @@
+// Copyright (c) Microsoft. All rights reserved.
+namespace DevOpsLib
+{
+    public enum BugPriority
```

Comment:
In DevOps, bug priority values can be 0-4.

Reply:
Create a class
1. private constructor with name and value properties.
2. create 3 static variables for each priority (one, two, other).
3. create a static method to return IEnumerable of these 3 static variables.
4. remove BugPriorityExtension

Reply:
I did it this way to avoid comparing non-enum types in the if else chain below.